# Dynamic Planner Default Params
main_run_freq                           : 2.5
voxel_dim                               : 0.15  # Unit: meter
robot_dim                               : 0.8   # Unit: meter
vehicle_height                          : 0.75  # Unit: meter
sensor_range                            : 30.0  # Unit: meter
terrain_range                           : 15.0  # Unit: meter
local_planner_range                     : 5.0   # Unit: meter
visualize_ratio                         : 0.75
is_viewpoint_extend                     : true
is_multi_layer                          : false
is_opencv_visual                        : false # Obstalce Image Display
is_static_env                           : true
is_pub_boundary                         : false
is_debug_output                         : false
is_attempt_autoswitch                   : true  # Auto switch to attemptable navigation
world_frame                             : map

# Graph Messager
GraphMsger/robot_id                     : 1  # graph from robot id "0" is extracted from files

# Map Handler Params
MapHandler/floor_height                 : 2.0     # Unit: meter
MapHandler/cell_length                  : 5.0     # Unit: meter
MapHandler/map_grid_max_length          : 1000.0  # Unit: meter
MapHandler/map_grad_max_height          : 100.0   # Unit: meter

# Dynamic Planner Utility Params
Util/angle_noise                        : 15.0 # Unit: degree
Util/accept_max_align_angle             : 4.0  # Unit: degree
Util/obs_inflate_size                   : 1
Util/new_intensity_thred                : 2.0
Util/terrain_free_Z                     : 0.2
Util/dyosb_update_thred                 : 4
Util/new_point_counter                  : 5
Util/dynamic_obs_dacay_time             : 10.0 # Unit: second
Util/new_points_decay_time              : 1.0  # Unit: second

# Dynamic Graph Params
Graph/connect_votes_size                : 10
Graph/clear_dumper_thred                : 4
Graph/node_finalize_thred               : 6
Graph/filter_pool_size                  : 12

# Corner Detector Params
CDetector/resize_ratio                  : 3.0
CDetector/filter_count_value            : 3
CDetector/is_save_img                   : false
CDetector/img_folder_path               : /path

# Graph Planner Params
GPlanner/converge_distance              : 0.5  # Unit: meter
GPlanner/goal_adjust_radius             : 2.0  # Unit: meter
GPlanner/free_counter_thred             : 7
GPlanner/reach_goal_vote_size           : 3
GPlanner/path_momentum_thred            : 3